_posts/2021-12-30-jmloyola-opensource-experience.md

```diff
@@ -0,0 +1,68 @@
+
+	The first project I got involved in and that I sent some patches to was [PyMC](https://github.com/pymc-devs/pymc). I found out about the project thanks to [Osvaldo Martin](https://github.com/aloctavodia), a researcher and core contributor of PyMC that works in the same research institute as I do. I attended a couple of his graduate courses and seminars where he used PyMC for probabilistic modeling. We talked about the project and he suggested a couple of ideas I could work on. A couple of days later I sent [my first pull request (PR) to PyMC](https://github.com/pymc-devs/pymc/pull/3389). I was excited, but the review process had just started. Although it looked like a simple PR, I realized that there is a lot of "extra" work you should do to get your code accepted. For example, discuss with other developers about the importance of the patch (sometimes after tinkering with the idea of the PR, the core developers realize that there is no need for the patch --that is why it is important to discuss it before starting to code), agree on a possible solution, and write tests and documentation for the changes. I said extra work, but actually, this is essential work needed to ensure the project is stable and easily understandable by the users.
+
+3. __How did you get involved in PyMC Google Summer of Code?__
+
+	I learned about Google Summer of Code (GSoC) from Osvaldo and a collegue, [Agustina Arroyuelo](https://github.com/agustinaarroyuelo). They worked in 2018 on the project ["Module for Approximate Bayesian Computation"](https://summerofcode.withgoogle.com/archive/2018/projects/5594656209371136). Agustina told me it was a great experience, and that motivated me to apply for the next year. In early 2019, Osvaldo and I started writing the proposal for the project ["Bayesian Additive Regression Trees in PyMC3"](https://summerofcode.withgoogle.com/archive/2019/projects/5713417321578496). While the proposal was being reviewed, I kept working on the project to improve my knowledge of the code-base and to get to know the rest of the core contributors. Fortunately, the project got accepted and we started working. Overall GSoC was a great experience and I will recommend it to anyone that wants to contribute to a particular project and can allocate some time for it.
+
+4. __How did you learn of the Data Umbrella LATAM sprint? Through any specific person?__
+
+	I found out about the Data Umbrella LATAM sprint 2021 through Twitter. I'm not an active social media user, but I do check Twitter once in a while. A month before the sprint I read a tweet from [Data Umbrella](https://twitter.com/DataUmbrella) in my feed inviting people to apply to the scikit-learn LATAM sprint. Since I wanted to get back to contributing to open source, I quickly applied.
```

Comment:
```suggestion
	I found out about the Data Umbrella LATAM sprint 2021 through [Twitter](https://twitter.com/DataUmbrella/status/1394661734275821573). I'm not an active social media user, but I do check Twitter once in a while. A month before the sprint I read a tweet from [Data Umbrella](https://twitter.com/DataUmbrella) in my feed inviting people to apply to the scikit-learn LATAM sprint. Since I wanted to get back to contributing to open source, I quickly applied.
```